test/conformance/chainsaw/verifyImages/clusterpolicy/standard/imageExtractors-complex-keyless/chainsaw-test.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   creationTimestamp: null
   name: imageextractors-complex-keyless
 spec:
+  timeouts:
+    apply: 10s
```

Comment:
If I understand correctly, this means that the test will timeout if apply does not finish in 10 seconds: https://github.com/kyverno/chainsaw/blob/8b2c70b6322cdb89c4e1bd45c5a26d631f150709/pkg/apis/v1alpha1/types.go#L125-L127

Since the policies for these tests have `webhookTimeoutSeconds: 30`, should this value be 30 seconds?